### DISCUSSION TITLE
Broke With Update - markdown and/or markers dependency update?

### DISCUSSION THREAD

**Discussion by JamesFraserLogan** [mkdocs/mkdocs#3627] [Q&A]
Hello, my name is James and I would like to share an issue I had this morning.

I couldn't run my mkdocs site this morning. It wouldn't run this:

```md
# My Module

::: modules.my_module
```

I checked my pipfile.lock and it said this had changed:
```ps1
@@ -243,7 +243,7 @@
                 "sha256:08695f5cb7ed6e0531a20572697297273c47b8cae5a63ffc6d6ed5c201be6e44",
                 "sha256:4f1d9991f5acc0ca119f9d443620b77f9d6b33703e51011c16baf57afb285fc6"
             ],
-            "markers": "sys_platform == 'win32'",
+            "markers": "platform_system == 'Windows'",
             "version": "==0.4.6"
         },
         "coverage": {
@@ -485,11 +485,11 @@
         },
         "markdown": {
             "hashes": [
-                "sha256:d43323865d89fc0cb9b20c75fc8ad313af307cc087e84b657d9eec768eddeadd",
-                "sha256:e1ac7b3dc550ee80e602e71c1d168002f062e49f1b11e26a36264dafd4df2ef8"
+                "sha256:48f276f4d8cfb8ce6527c8f79e2ee29708508bf4d40aa410fbc3b4ee832c850f",
+                "sha256:ed4f41f6daecbeeb96e576ce414c41d2d876daa9a16cb35fa8ed8c2ddfad0224"
             ],
             "markers": "python_version >= '3.8'",
-            "version": "==3.5.2"
+            "version": "==3.6"
         },
```
     
The fix is to not update packages; I am not which package is responsible or if it's really your problem, or if this is more of a problem with griffe. But if it is a bug that effects your great package that I love so much, I hope this helps.

I am running python 3.11.6 on windows 11 pro 64 bit.

**Comment by pawamoy** [comment]
Hello @JamesFraserLogan, you'll have to actually explain what the problem is haha :smile: 
I'll move this to Q/A for now (likely unrelated to MkDocs itself) :slightly_smiling_face:
